news/2017-10-28-beatmap-spotlights-september-2017.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Beatmap Spotlights September 2017"
+date: 2017-10-28 16:00:00 +0000
+---
+
+Here we are again, introducing a selection of the best maps of the previous month. Come take a look at September’s Beatmap Spotlights, and snag yourself a shiny new medal in the process!
```

Comment:
make explicit reference to September since September isn't "last month" anymore

Reply:
Did both, kinda unsure about the opener tho feels kinda meh to me now.